src/api/app/components/bs_request_activity_timeline_component.html.haml

```diff
@@ -1,9 +1,13 @@
-- timeline.each do |comment_or_history_element|
-  .timeline-item.mb-3
-    - if comment_or_history_element.is_a?(Comment)
-      .comments-thread
-        = render BsRequestCommentComponent.new(comment: comment_or_history_element, level: 1, commentable: comment_or_history_element.commentable,
-                                               diff: diff_for_ref(comment_or_history_element))
-    - else
-      = render BsRequestHistoryElementComponent.new(element: comment_or_history_element,
-                                                    request_reviews_for_non_staging_projects: request_reviews_for_non_staging_projects)
+- if timeline.empty? && timeline_for_superseeded
```

Comment:
Can't we just check `timeline.empty?` and show `No activity`? What is the reason of introducing `timeline_for_superseeded`?

Reply:
The component is used for the `timeline` of the current request too (https://github.com/openSUSE/open-build-service/pull/17895/files/0a97068568915ff6197c2f65546130a7f366eab0#diff-55c954c31d3d0da8d443fde4a6a28c763231badacdc004fc36ae07703659bf9eR78). I find odd to have `No activity.` for the current timeline. Would you prefer to see that in there too?

Reply:
ah I see. The component is used twice. Got it